examples/apps/color_finder.py

```diff
@@ -88,6 +90,30 @@ def color_distance_sqr(color, color_range):
         bdist_sq = (maxB-b)**2
     return rdist_sq+gdist_sq+bdist_sq
 
+def gray_world(img):
```

Comment:
I think this name is confusing (I'd expect it to turn the image to grayscale), so even if that's the name of the algorithm i think it would be good to rename this more clearly - e.g. something related to what it's doing (adjusting the image color balance to average to gray?)